README_FIRST

```diff
@@ -1,3 +1,9 @@
+The master repository for manage_externals is
```

Comment:
This file should actually be removed, it was the strawman for the doc that went into the cesm readme.

Reply:
So should this go in the output from `--help`, which gets put in README.md? Or do you have some other suggestion for where to put this information?